src/particles/elements/diagnostics/openPMD.cpp

```diff
@@ -253,6 +254,17 @@ namespace detail
                 getComponentRecord(component_name).resetDataset(d_fl);
             }
         }
+
+        // openPMD coarse position
+        {
+            beam["positionOffset"]["x"].resetDataset(d_fl);
+            beam["positionOffset"]["x"].makeConstant(ref_part.x);
+            beam["positionOffset"]["y"].resetDataset(d_fl);
+            beam["positionOffset"]["y"].makeConstant(ref_part.y);
+            beam["positionOffset"]["ct"].resetDataset(d_fl);
+            beam["positionOffset"]["ct"].makeConstant(ref_part.t);
```

Comment:
@cemitch99 here and in position, do we want to output `s` instead of `ct`?

Reply:
Not sure I follow what the intended output is here.  I understand that by "ct" you mean "ref_part.t", but what is meant by "dt"?  Does this represent one or more particle values of the "t" coordinate?

In general, I would say that "s" is qualitatively different from the other variables like ct.  It is an independent variable, it is associated only with the reference particle, and the other (beam particle) values are given as functions of "s".

Reply:
Thanks, ah yes I was confused for a second... So per particle, splitting the longitudinal position (for a fixed s) into `ref_part.t` (should we call this `ref_part.ct`?) and `position.ct` makes sense...

Reply:
Yes, that's true.

Reply:
Ok, will do in #350